conf/mariadb/mariadb.conf.tt.example

```diff
@@ -0,0 +1,160 @@
+#
+# These groups are read by MariaDB server.
+# Use it for options that only the server (but not clients) should see
+#
+# See the examples of server my.cnf files in /usr/share/mysql/
+#
+
+# this is read by the standalone daemon and embedded servers
+[server]
+
+# this is only for the mysqld standalone daemon
+[mysqld]
+
+port    = 3306
+socket    = /var/lib/mysql/mysql.sock
+skip-external-locking
+key_buffer_size = [% key_buffer_size %]M
+max_allowed_packet = 16M
+table_open_cache = 512
+sort_buffer_size = 2M
+read_buffer_size = 2M
+read_rnd_buffer_size = 8M
+myisam_sort_buffer_size = 64M
+thread_cache_size = 10
```

Comment:
I suggest making this configurable.
10 may not be enough for everyone.
Since it's so cheap, I would at least raise the default to 25 or even 50.